### PR TITLE
Tests sign up

### PR DESCRIPTION
**Pull request by manueltuero**
## Summary
Add tests for `users` service

## Trello Card
https://trello.com/c/ZFOyg5LT

### CODE REVIEW

File: app/services/users.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const { users } = require('../models'),
+  logger = require('../logger'),
+  errors = require('../errors'),
+  bcrypt = require('bcryptjs');
+const saltRounds = 10;
+
+const haveAllProps = user => user.firstName && user.lastName && user.email && user.password;
```
Comment (jersonlopez) [inline app/services/users.js]: What do you think about express-validator?

File: migrations/migrations/20190628184513-create-user.js [hunk 2]
```diff
@@ -25,16 +25,6 @@ module.exports = {
       password: {
         allowNull: false,
         type: Sequelize.STRING
-      },
```
Comment (jersonlopez) [inline migrations/migrations/20190628184513-create-user.js]: Why did you remove these fields?

File: package.json [hunk 3]
```diff
@@ -17,7 +18,10 @@
     "lint-fix": "eslint \"**/*.js\" --fix --ignore-pattern ./.eslintrc.js",
     "precommit": "npm run lint-diff",
     "outdated": "npm outdated --depth 0",
-    "pretest": "npm run lint",
+    "create-dev-db": "sudo -i -u postgres psql -c 'CREATE DATABASE db_wtraining_dev;' -U postgres",
```
Comment (jersonlopez) [inline package.json]: all these scripts aren't necessary

Comment (manueltuero) [reply package.json]: Why not?

File: app/services/users.js (hunk 1 above)
Comment (manueltuero) [reply app/services/users.js]: I think that is great! I will implement it in the next PR (middlewares)

File: migrations/migrations/20190628184513-create-user.js (hunk 2 above)
Comment (manueltuero) [reply migrations/migrations/20190628184513-create-user.js]: Because I think that is not necessary for the training. Maybe I'm probably wrong.
